packages/snapshot-listener/src/config.ts

```diff
@@ -1,11 +1,8 @@
-import * as dotenv from 'dotenv';
-dotenv.config();
+import { config as target } from '@hicommonwealth/adapters';
 
-export const NODE_ENV = process.env.NODE_ENV || 'development';
-export const RABBITMQ_URI = (() => {
-  if (!process.env.CLOUDAMQP_URL || process.env.NODE_ENV === 'development') {
-    return 'amqp://127.0.0.1';
-  } else return process.env.CLOUDAMQP_URL;
-})();
+const { PORT } = process.env;
 
-export const DEFAULT_PORT = '8001';
+export const config = {
+  ...target,
+  PORT: parseInt(PORT ?? '8001', 10),
```

Comment:
This will break local development if `PORT` is defined in the `.env` file. `PORT` can only be defined once in the `.env` file so if you try to start 2 different services that use `PORT` it will break. We may want to ensure `PORT` is not defined in `.env` if `NODE_ENV !=== 'production'`. Not a huge issue but worth noting.

Reply:
When I was writing this, was thinking that maybe we need a "WORKERS" config section with ports (and probably other specific vars) to support this scenario.... something like PORT: WORKERS.SNAP.PORT ?? PORT

Reply:
but probably better to hardcode 8001 when development, and PORT otherwise

Reply:
Yea I like the second option: `if (NODE_ENV !== 'production'): PORT = 8001 else PORT = config.PORT`